app/ts/main/DebugMonitor.ts

```diff
@@ -55,19 +71,4 @@ export class DebugMonitor {
       this.values[index] = value
     }
   }
-
```

Comment:
removed this code as it was redundant (calling updateField in a loop would have the same effect only be more efficient and readable)